test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="php-fpm-client-$RANDOM-$RANDOM"
+docker build -q -t "$client_image" - > /dev/null <<EOF
```

Comment:
Got it. Aside from some naming convention for test related container that the top-level `run.sh` could clean up, I'm not sure what can be done about that. Container tagging/metadata would be a nice to have for this, assuming Docker ever gets it.

FWIW, building this client image is pretty quick since it's just installing a single package. Perhaps since this image doesn't need to be really tiny, there could be a base image for test client container that has already run `apt-get update`. That way the client install would be the only thing to wait for.

Just throwing out ideas. Not sure what the right approach is.